openupgrade_scripts/scripts/purchase/14.0.1.2/pre-migration.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2021 ForgeFlow S.L.  <https://www.forgeflow.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+def fill_purchase_order_line_qty_to_invoice(env):
+    if not openupgrade.column_exists(env.cr, "purchase_order_line", "qty_to_invoice"):
+        openupgrade.logged_query(
+            env.cr,
+            """
+            ALTER TABLE purchase_order_line
+            ADD COLUMN qty_to_invoice integer""",
```

Comment:
Believe this should be float not integer

Reply:
done, thanks